control/trajectory_follower/lane_departure_checker/include/lane_departure_checker/lane_departure_checker_node.hpp

```diff
@@ -0,0 +1,116 @@
+// Copyright 2020 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef LANE_DEPARTURE_CHECKER__LANE_DEPARTURE_CHECKER_NODE_HPP_
+#define LANE_DEPARTURE_CHECKER__LANE_DEPARTURE_CHECKER_NODE_HPP_
+
+#include <memory>
+#include <vector>
+
+#include "autoware_utils/ros/debug_publisher.hpp"
+#include "autoware_utils/ros/processing_time_publisher.hpp"
+#include "autoware_utils/ros/self_pose_listener.hpp"
+#include "diagnostic_updater/diagnostic_updater.hpp"
+#include "lanelet2_core/LaneletMap.h"
+#include "lanelet2_extension/utility/message_conversion.hpp"
+#include "rclcpp/rclcpp.hpp"
+
+#include "autoware_planning_msgs/msg/route.hpp"
+#include "autoware_planning_msgs/msg/trajectory.hpp"
+#include "geometry_msgs/msg/pose_stamped.hpp"
+#include "std_msgs/msg/float64.hpp"
```

Comment:
std_msgs is deprecated.
Can you use autoware_debug_msgs/msg/Float64Stamped instead?

Reply:
fixed